.github/workflows/release.yml

```
@@ -17,46 +22,63 @@
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0 # required for getting commit SHA
+          fetch-depth: 0 # required for getting commit history
 
-      - name: Generate version string
+      - name: Set up build cache
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)